system-tests/__snapshots__/record_spec.js

```diff
@@ -489,7 +489,7 @@ exports['e2e record passing passes 2'] = [
                 'originalFile': 'cypress/e2e/record_uncaught.cy.js',
                 'relativeFile': 'cypress/e2e/record_uncaught.cy.js',
                 'absoluteFile': '/foo/bar/.projects/e2e/cypress/e2e/record_uncaught.cy.js',
-                'frame': '> 1 | throw new Error(\'instantly fails\')\n    |       ^\n  2 | ',
+                'frame': '> 1 | throw new Error(\'instantly fails\')\n    |       ^\n  2 |',
```

Comment:
snapshot update likely due to the babel code frame updates (change was a space).